hugo/layouts/partials/home/hero.html

```diff
@@ -0,0 +1,27 @@
+<header class="home-hero">
+    <div class="home-hero__wrap">
+        <div class="home-hero__info">
+            <h1>{{ .title }}</h1>
+            <p>{{ .desc }}</p>
+            <div class="home-hero__info__links">
```

Comment:
Do we have different documents?

![screenshot_2017-06-08_13 41 59](https://user-images.githubusercontent.com/1312023/26926997-4d82a550-4c50-11e7-8d50-8fc2538316ce.png)
![screenshot_2017-06-08_13 42 08](https://user-images.githubusercontent.com/1312023/26926998-4d835b62-4c50-11e7-9dcb-74c4cd7baee5.png)